agent/package.json

```
@@ -13,7 +13,8 @@
   "scripts": {
     "build:root": "pnpm -C .. run -s build",
     "build:agent": "node src/esbuild.mjs",
-    "build": "pnpm run -s build:root && pnpm run -s build:agent",
+    "build:webviews": "pnpm -C ../vscode _build:webviews --mode production --outDir $PWD/dist/webviews",
```

```suggestion
    "build:webviews": "pnpm -C ../vscode _build:webviews --mode production --outDir ../../agent/dist/webviews",
```

Running this created a folder in my vscode directory called "$PWD/dist/webviews" 🤣